README.md

```diff
@@ -10,6 +10,7 @@ In this repository, you can find the [official GitHub public feedback discussion
 | 👩‍✈️  [Copilot](https://github.com/github/feedback/discussions/categories/copilot-feedback)   	| [GitHub Copilot](https://copilot.github.com/) (Technical Preview) 	|
 | 🤖  [Dependabot](https://github.com/github/feedback/discussions/categories/dependabot-feedback) 	| [GitHub Dependabot](https://github.com/features/security) 	|
 | 🗣️  [Discussions](https://github.com/github/feedback/discussions/categories/discussions-feedback)  	| [GitHub Discussions](https://docs.github.com/en/discussions) 	|
+| 🌐  [Feed Feedback](https://github.com/github/feedback/discussions/categories/feed-feedback)  	| [GitHub Feed Feedback](https://github.blog/2022-03-22-improving-your-github-feed/) 	|
```

Comment:
Drop the word Feedback from both cells in the table

Reply:
👍🏾